requirements.txt

```
@@ -1,6 +1,6 @@
 mkdocs==1.5.3
```

We can now apply [this update](https://github.com/codacy/pulse-user-docs/pull/290/files).

```suggestion
mkdocs==1.6.0
```